cluster/manifests/prometheus/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
does it work, because we pull API via extensions from skipper and kube-ingress-aws-controller, too IIRC?

Reply:
It works because the resources are served via both APIs. `kubectl get ingress` will currently use `extensions/v1beta1` by default.